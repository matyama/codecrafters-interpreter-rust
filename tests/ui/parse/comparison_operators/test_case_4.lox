(24 - 98) >= -(27 / 34 + 17)
