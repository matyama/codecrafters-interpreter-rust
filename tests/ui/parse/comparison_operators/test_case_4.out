(>= (group (- 24.0 98.0)) (- (group (+ (/ 27.0 34.0) 17.0))))
